src/components/DiffView/index.spec.tsx

```diff
@@ -97,4 +112,69 @@ describe(__filename, () => {
     });
     expect(_tokenize).toHaveBeenCalledTimes(parsedDiffs.length);
   });
+
+  it('configures anchors/links on each line number', () => {
+    const root = render();
+
+    expect(root.find(Diff)).toHaveProp('gutterType', 'anchor');
+    // More info about the `getChangeKey()` function here: https://github.com/otakustay/react-diff-view/tree/6aa5399c52392e19f7f8fbe4af17b374b4339862#key-of-change
+    expect(root.find(Diff)).toHaveProp('generateAnchorID', getChangeKey);
+  });
+
+  it('passes a selected change from the location hash to the Diff component', () => {
+    // The value is usually generated by `getChangeKey()`.
+    const selectedChange = 'I1';
+    const location = createFakeLocation({ hash: `#${selectedChange}` });
+
+    const root = render({ location });
+
+    expect(root.find(Diff)).toHaveProp('selectedChanges', [selectedChange]);
+  });
+
+  it('does not pass any selected changes when location hash is empty', () => {
+    const location = createFakeLocation({ hash: '' });
+
+    const root = render({ location });
+
+    expect(root.find(Diff)).toHaveProp('selectedChanges');
```

Comment:
This spec suggests that you meant to check that the prop is empty, right? Something like this, maybe?

```diff
diff --git a/src/components/DiffView/index.spec.tsx b/src/components/DiffView/index.spec.tsx
index f2aba68..04a951a 100644
--- a/src/components/DiffView/index.spec.tsx
+++ b/src/components/DiffView/index.spec.tsx
@@ -136,7 +136,7 @@ describe(__filename, () => {
 
     const root = render({ location });
 
-    expect(root.find(Diff)).toHaveProp('selectedChanges');
+    expect(root.find(Diff)).toHaveProp('selectedChanges', undefined);
   });
 
   it('tries to find the selected line element on mount', () => {
```

Reply:
ah yes, it'll be more explicit indeed

Reply:
I changed to empty array because passing `undefined` to `Diff` results in a prop set to `[]` (default value) so I set this default value in our code too.